### ISSUE TITLE
msgid 'Continue reading %s' still has the issue like the issue #382

### ISSUE THREAD

**Issue by tg29359** [WordPress/twentyseventeen#420]
The %s is screen reader text, and it is also too difficult to translate to Japanese like the issue #382.
Please grep whole source code with the key word "screen-reader-text", and make a search for "%s" around the "screen-reader-text", then consider whether the %s should be wrapped with `<span>` or not.
